ldclient/feature_store.py

```diff
@@ -10,24 +10,24 @@ def __init__(self):
         self._initialized = False
         self._features = {}
 
-    def get(self, key):
+    def get(self, key, callback):
         try:
             self._lock.rlock()
             f = self._features.get(key)
             if f is None:
                 log.debug("Attempted to get missing feature: " + str(key) + " Returning None")
-                return None
+                return callback(None)
             if 'deleted' in f and f['deleted']:
                 log.debug("Attempted to get deleted feature: " + str(key) + " Returning None")
-                return None
-            return f
+                return callback(None)
+            return callback(f)
```

Comment:
I'm probably missing something, but is this class used by both Twisted and non-Twisted? 

The interface docs below state 

> `:return: The feature value. None if not found`

So shouldn't this execute `callback(f)` but `return f`?

And if I'm understanding this correctly, then I think you can give the callback parameter a default value (of a no-op function) to lessen the impact of the API changes.